lib/ddtrace/debug/health.rb

```diff
@@ -0,0 +1,31 @@
+require 'ddtrace/ext/debug'
+
+module Datadog
+  module Debug
+    # Health-related debugging
+    module Health
+      # Health metrics for debugging
+      class Metrics < ::Datadog::Metrics
+        distribution :api_errors, Ext::Debug::Health::Metrics::METRIC_API_ERRORS
```

Comment:
we probably just want these as counters (increment)

Reply:
or well, that is what we are doing in Python, haven't figured out if we should submit as different metric types or not yet.

Reply:
I remember someone who worked on Metrics said distributions should always be used (over counts.) Hence, why these are distributions.

Reply:
Distributions are a feature still in beta (according to our docs), so they should not be used here.

https://docs.datadoghq.com/developers/metrics/metrics_type/?tab=distribution#metric-type-definition

Reply:
Okay, we don't have counts in Ruby, so I guess we can switch it to increment.

Reply:
Never mind, have to add `count` because have to increment by arbitrary amounts in some cases.

Reply:
Pretty sure it does. I'm going to adjust it